.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
That seems too easy to get wrong by accident.  We have the same problem though if someone wanted settings for their editor of choice, so where do we draw the line?